minemeld/flask/sns.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import json
+import os.path
+import uuid
+import minemeld
+from .logger import LOG
+from . import config
+
+SNSAPIURL = config.get('SNS_URL', 'https://o11kiw0vpe.execute-api.us-east-1.amazonaws.com/run')
+SNSENABLED = config.get('SNS_ENABLED', False)
+UUIDFILENAME = config.get('UUID_FILE', 'uu.id4')
+TYPEHELLO = 'hello'
+TYPEMKWISH = 'mkwish'
+TYPESTATS = 'stats'
+
+sns_obj = {}
+sns_available = False
+
+
+class sns:
```

Comment:
Could you:
- capitalize the class name
- inherit from `object` ? See https://docs.python.org/release/2.2.3/whatsnew/sect-rellinks.html

Reply:
OK